# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../uclibc/pkg_patch/openssh/susv3_legacy.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

No idea why -D__UCLIBC_SUSV3_LEGACY_MACROS__ does not work

--- ./includes.h.orig	2010-05-27 12:05:28.000000000 -0400
+++ ./includes.h	2010-05-27 12:04:42.000000000 -0400
@@ -58,6 +58,7 @@
  *-*-nto-qnx needs these headers for strcasecmp and LASTLOG_FILE respectively
  */
 #ifdef HAVE_STRINGS_H
+# define __UCLIBC_SUSV3_LEGACY_MACROS__
 # include <strings.h>
 #endif
 #ifdef HAVE_LOGIN_H
